### ISSUE TITLE
Unable to use syslog as input pipeline, not starting

### ISSUE THREAD

**Issue by whysi** [fluent/fluent-bit#5153]
## Bug Report

**Unable to use syslog as input pipeline, not starting**

I'm trying to start fluent-bit with the syslog input using various type of installations under Windows. All of them are failing with:
`[error] [config] section 'syslog' tried to instance a plugin name that don't exists`

**To Reproduce**
- Clean installation of Fluent-Bit on Windows (x64), tested with version 1.9.0-win64 and 1.8.13-win64 or just use the .zip archive
-  Start a simple execution with syslog as input and stdout as output, via command line (`fluent-bit.exe -i syslog -o stdout -R parsers.conf`) or with a simple config file, like this:
 ```
[SERVICE]
    Flush         1
    Log_Level     trace
    Daemon        Off
    Parsers_File  parsers.conf
    HTTP_Server   Off
    HTTP_Listen   0.0.0.0
    HTTP_Port     2020

[INPUT]
    Name              syslog
    Parser            syslog-rfc3164
    Mode              tcp
    Listen            0.0.0.0
    Port              24225
    Buffer_Chunk_Size 32KB
    Buffer_Max_Size   64KB

[OUTPUT]
    Name             stdout
    Match            *
```

The parsers file is the default one shipped with the installation/archive.

**Expected behavior**

Application starting as expected with syslog input using the given parsers.

**Screenshots**

![image](https://user-images.githubusercontent.com/93594835/159498009-df1b3a93-9bf4-47cb-bcfb-08b18790e04f.png)

Error log result:
`[2022/03/22 14:56:00] [error] [config] section 'syslog' tried to instance a plugin name that don't exists
[2022/03/22 14:56:00] [error] configuration file contains errors, aborting.`

**Your Environment**

* Version used: 1.9.0-win64 and 1.8.13-win64
* Configuration: 
```
[SERVICE]
    Flush         1
    Log_Level     trace
    Daemon        Off
    Parsers_File  parsers.conf
    HTTP_Server   Off
    HTTP_Listen   0.0.0.0
    HTTP_Port     2020

[INPUT]
    Name              syslog
    Parser            syslog-rfc3164
    Mode              tcp
    Listen            0.0.0.0
    Port              24225
    Buffer_Chunk_Size 32KB
    Buffer_Max_Size   64KB

[OUTPUT]
    Name             stdout
    Match            *
```
* Environment: Windows 10, Windows Server 2022 core, Windows Server 2022 nano docker image

**Comment by nokute78** [comment]
According to build configuration , fluent-bit doesn't contain in_syslog on Windows system.
https://github.com/fluent/fluent-bit/blob/v1.9.0/cmake/windows-setup.cmake#L30

**Comment by whysi** [comment]
> According to build configuration , fluent-bit doesn't contain in_syslog on Windows system. https://github.com/fluent/fluent-bit/blob/v1.9.0/cmake/windows-setup.cmake#L30

I've missed that, thank you very much for your help!

**Comment by sean-scott-lr** [comment]
Not sure why this was closed.  Why doesnt in_syslog work on windows? Is it an oversight, or purposeful?

**Comment by tirelibirefe** [comment]
the same problem exist also for exec plugin.

`[error] [config] section 'exec' tried to instance a plugin name that don't exists`

**Comment by leonardo-albertovich** [comment]
Could you please create a new issue with a proper report of your issue so we can give you a meaningful answer?

**Comment by leonardo-albertovich** [comment]
@sean-scott-lr : `in_syslog` wasn't enabled in windows because it optionally uses a unix socket to receive data and not making just that feature platform dependent was an oversight. We have already made the necessary changes to enable its usage in windows and it will be included in the next release of fluent-bit 2.1.
